joomla/joomla-cms
# [com_fields] Adding label Class for input

Pull Request for Issue #16994.
## Summary of Changes
Removed the hide Label Option Replaced hide Label Option with a label Class Option. Added B/C Option to add class hidden to the class if label was set previously to hidden. Added a label class Option for Form Reordered the Options and added headlines.
See also discussion in #17177
## Testing Instructions
	1.	Hide your label, apply the patch and see if the label is still hidden
	2.	Add a class for the label in the form and see if it's added
	3.	Add a class for the label in the output rendering and see if it's added
## Documentation Changes Required
Yes adding the possibility to add a Label Class in the form.

<img width="813" alt="grafik" src="https://user-images.githubusercontent.com/828371/32124636-cb3e657e-bb68-11e7-9772-ce2dd4c7abcb.png">

![grafik](https://user-images.githubusercontent.com/828371/32124646-d3114758-bb68-11e7-9b55-0f8a1e13d6cb.png)
